libs/community/langchain_community/chat_models/baidu_qianfan_endpoint.py

```diff
@@ -103,7 +103,7 @@ class QianfanChatEndpoint(BaseChatModel):
     streaming: Optional[bool] = False
     """Whether to stream the results or not."""
 
-    request_timeout: Optional[int] = 60
+    timeout: Optional[int] = 60
```

Comment:
need to make sure we do this in a backwards compatible way:
```python
request_timeout: Optional[int] = Field(default=60, alias="timeout")
```
see https://github.com/langchain-ai/langchain/pull/20161 for an example

and we should add a unit test to make sure current and new init args work

Reply:
> need to make sure we do this in a backwards compatible way:
> 
> ```python
> request_timeout: Optional[int] = Field(default=60, alias="timeout")
> ```
> 
> see #20161 for an example
> 
> and we should add a unit test to make sure current and new init args work

I see.Thanks for review.